FAQ.md

```diff
@@ -0,0 +1,61 @@
+# TrebleShot
+
+##### What is the device name?
+
+The device name is the name used to identify TrebleShot users.
+
+##### What's my default TrebleShot name? 
+
+The default TrebleShot name is the name of your device.
+
+##### What permission does TrebleShot require?
+
+TrebleShot requires the following permission on your device:
```

Comment:
How about this: TrebleShot needs to access these on your device (requirements may vary):

Reply:
Cool,  I would edit it now.